frontend/www/js/omegaup/arena/arena.js

```diff
@@ -418,6 +419,9 @@ export class Arena {
         data: {
           problems: [],
           activeProblem: null,
+          inAssignment:
+            typeof self.options.courseAlias !== 'undefined' &&
+            self.options.courseAlias !== null,
```

Comment:
```suggestion
            !!self.options.courseAlias,
```

eso se encarga de checar que no es nulo, `undefined` Y que no está vacío.

Reply:
Listo